presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestMergeIdenticalWindows.java

```diff
@@ -42,18 +41,24 @@
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.functionCall;
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.join;
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.project;
+import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.symbolStem;
 import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.window;
 import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
 
 public class TestMergeIdenticalWindows
 {
     private final LocalQueryRunner queryRunner;
-    private final Window windowA;
-    private final Window windowB;
+    private final Symbol suppkey;
+    private final Symbol orderkey;
+    private final Symbol shipdate;
+    private final Optional<WindowFrame> commonFrame;
```

Comment:
Isn't this change included already in `window_refactor/2` or `window_multiple_frames/2` already?